src/aerovaldb/sqlitedb/sqlitedb.py

```diff
@@ -616,3 +619,34 @@ def list_timeseries(
 
             uri = build_uri(route, route_args, kwargs)
             yield uri
+
+    def rm_experiment_data(self, project: str, experiment: str) -> None:
+        cur = self._con.cursor()
+        for table in [
+            "glob_stats",
+            "contour",
+            "timeseries",
+            "timeseries_weekly",
+            "config",
+            "menu",
+            "statistics",
+            "ranges",
+            "regions",
+            "models_style0",
+            "map0",
+            "map1",
+            "scatter0",
+            "scatter1",
+            "profiles",
+            "heatmap_timeseries0",
+            "heatmap_timeseries1",
+            "heatmap_timeseries2",
+            "forecast",
+            "gridded_map",
+        ]:
+            cur.execute(
+                f"""
+                DELETE FROM {table} WHERE project=? AND experiment=?
+                """,
+                (project, experiment),
+            )
```

Comment:
I would expect `rm_experiment_data` to remove all experiment data, including the entry in `experiments.json`.

Reply:
I think that as long as aerovaldb isn't responsible for writing the experiments file (which it doesn't currently do), it shouldn't remove anything from this file either. There is a whole bunch of file-synchronization code in pyaerocom ExperimentOutput that keeps files in sync, and I don't think splitting that logic into two repos is an improvement over the current state.

That being said. I like the idea of having experiments be a 'virtual' query which is generated based on the other data files present in the db.